az-digital/arizona-bootstrap
# Closes #213 - Add "body freeze" when mobile nav is open

## Description

Closes #213 by adding a "body freeze" when the offcanvas mobile nav is open, so that you can't scroll through the background while you have the mobile nav open.

Closes #247 

Review site: https://review.digital.arizona.edu/arizona-bootstrap/issue/213/docs/2.0/components/navbar-off-canvas/

## Review Instructions
- View the Offcanvas Nav component
- Resize browser to mobile
- Open Offcanvas nav
   - Ensure background doesn't scroll
   - Ensure mobile nav still scrolls
